UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/OverlayMenu.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using MapFeatures.Overlays;
+
+/* 
+ * Allows the user to choose when to display the grid and how big it should be.
```

Comment:
not generic enough - right now it's the grid but the menu is Overlay in general?

Reply:
correct, will change this